src/LinkedList/linked_list.c

```diff
@@ -28,6 +28,10 @@ pLinkedList makeList() {
   // Allocate enough space for a linked list, but malloc return a (void*)
   // We need to cast it to a pLinkedList.
   pLinkedList linkedList = (pLinkedList) malloc(sizeof(LinkedList));
+  if (linkedList == NULL) {
+    printf("Malloc failed\n");
+    return 0;
```

Comment:
Although NULL evaluates to 0 (in most systems), lets return the NULL macro so that in other systems where non-0 NULL is represented our library would work.